frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelTab.tsx

```diff
@@ -12,10 +12,18 @@ import { TestAccountFilter } from 'scenes/insights/TestAccountFilter'
 import { InsightTitle } from '../InsightTitle'
 import { SaveOutlined } from '@ant-design/icons'
 import { isValidPropertyFilter } from 'lib/components/PropertyFilters/utils'
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+import { ToggleButtonChartFilter } from './ToggleButtonChartFilter'
+import { InsightActionBar } from '../InsightActionBar'
+import { preflightLogic } from 'scenes/PreflightCheck/logic'
 
 export function FunnelTab(): JSX.Element {
     useMountedLogic(funnelCommandLogic)
     const { isStepsEmpty, filters, stepsWithCount } = useValues(funnelLogic())
+    const { featureFlags } = useValues(featureFlagLogic)
+    const { preflight } = useValues(preflightLogic)
+    const autoCalculate = featureFlags[FEATURE_FLAGS.FUNNEL_BAR_VIZ] && preflight?.is_clickhouse_enabled // Query performance is suboptimal on psql
```

Comment:
This is the second place. Could be a selector from `funnelLogic` for convenience